telegram/inline/inlinekeyboardmarkup.py

```diff
@@ -53,6 +53,12 @@ def __init__(self, inline_keyboard: List[List[InlineKeyboardButton]], **_kwargs:
 
         self._id_attrs = (self.inline_keyboard,)
 
+        if not self._check_keyboard_type(inline_keyboard):
+            raise ValueError(
+                "The parameter `inline_keyboard` should be a list of "
+                "list of InlineKeyboardButtons"
+            )
```

Comment:
Do we want to throw an error or just a warning? 🤔

Reply:
if we warn then its going to raise an error later anyway, might as well do it earlier with a proper message

Reply:
mh, yeah … makes sense :)